packages/units/src.ts/index.ts

```diff
@@ -64,9 +64,9 @@ export function commify(value: string | number): string {
 export function formatUnits(value: BigNumberish, unitName?: string | BigNumberish): string {
     if (typeof(unitName) === "string") {
         const index = names.indexOf(unitName);
-        if (index !== -1) { unitName = 3 * index; }
+        if (index !== -1) { unitName = Math.max((3 * index) - 1, 0); }
```

Comment:
Wouldn't it be easier if we simply use `unitName = 2 * index` ?

Reply:
It's not the same, and the calculations would be wrong, based on [https://docs.hedera.com/guides/docs/sdks/hbars](https://docs.hedera.com/guides/docs/sdks/hbars)
```
✅ (3*0)-1 = 0   =>  tinybar (because of Math.max)
✅ (3*1)-1 = 2   =>  microbar = 10^2 tinybars (100)
✅ (3*2)-1 = 5   =>  millibar = 10^5 tinybars (100,000)
✅ (3*3)-1 = 8   =>  hbar = 10^8 tinybars (100,000,000)
✅ (3*4)-1 = 11  =>  kilobar = 10^11 tinybars (100,000,000,000)
✅ (3*5)-1 = 14  =>  megabar = 10^14 tinybars (100,000,000,000,000)
✅ (3*6)-1 = 17  =>  gigabar = 10^17 tinybars (100,000,000,000,000,000)

✅ 2*0 = 0   =>  tinybar
✅ 2*1 = 2   =>  microbar = 10^2 tinybars (100)
❌ 2*2 = 4   =>  millibar = 10^4 tinybars (10,000)
❌ 2*3 = 6   =>  hbar = 10^6 tinybars (1,000,000)
❌ 2*4 = 8   =>  kilobar = 10^8 tinybars (100,000,000)
❌ 2*5 = 10  =>  megabar = 10^10 tinybars (10,000,000,000)
❌ 2*6 = 12  =>  gigabar = 10^12 tinybars (1,000,000,000,000)
```